cap/modules/schemas/cli.py

```diff
@@ -30,11 +30,112 @@
 from flask_cli import with_appcontext
 from invenio_db import db
 from invenio_jsonschemas.errors import JSONSchemaNotFound
+from invenio_search import current_search_client
 from sqlalchemy.exc import IntegrityError
 
+from cap.cli import MutuallyExclusiveOption
+from cap.modules.deposit.errors import DepositValidationError
 from cap.modules.fixtures.cli import fixtures
+from cap.modules.records.api import CAPRecord
+from cap.modules.schemas.models import Schema
+from cap.modules.schemas.resolvers import resolve_schema_by_url,\
+    resolve_schema_by_name_and_version, schema_name_to_url
 
-from .models import Schema
+
+@fixtures.command()
```

Comment:
change command to `records validate ..`

Reply:
we need to check if we can update the group commands